app/views/courses/TeacherClassView.coffee

```diff
@@ -500,7 +508,11 @@ module.exports = class TeacherClassView extends RootView
           .replace('{{remainingSpots}}', remainingSpots)
         )
       noty text: lines.join('<br />'), layout: 'center', type: 'information', killer: true, timeout: 5000
-    
+
+      # TODO: does not reflect existing student progress from outside of this classroom
```

Comment:
I strongly prefer phrasing TODO's as tasks; a lot of the time it's unclear what needs doing based on statement-of-fact TODOs.